gcsfs/core.py

```diff
@@ -296,13 +329,47 @@ def _call(self, method, path, *args, **kwargs):
         meth = getattr(requests, method)
         if args:
             path = path.format(*[quote_plus(p) for p in args])
-        r = meth(self.base + path, headers=self.header, params=kwargs,
-                 json=json)
+        for retry in range(RETRY_COUNT):
+            try:
+                time.sleep(2**retry-1)
+                r = meth(self.base + path, headers=self.header, params=kwargs,
+                         json=json)
+                validate_response(r, path)
+                break
+            except RuntimeError as e:
+                warnings.warn("_call caught RuntimeError: " + e.__repr__())
+                with open('/tmp/error', 'wb') as f:
```

Comment:
We definitely don't want to do this, even less so to an absolute path that might not exist or be writable. I would say log in the appropriate places (sparingly) and then the user can always attach an arbitrary log handler to do whatever they like with the messages.

Reply:
Oh yes sorry, that's debug code I forgot to remove!